packages/frontend/editor-ui/src/components/WhatsNewModal.vue

```diff
@@ -0,0 +1,264 @@
+<script setup lang="ts">
+import {
+	DynamicScroller,
+	DynamicScrollerItem,
+	type RecycleScrollerInstance,
+} from 'vue-virtual-scroller';
+import 'vue-virtual-scroller/dist/vue-virtual-scroller.css';
+import dateformat from 'dateformat';
+import { useI18n } from '@n8n/i18n';
+import { VERSIONS_MODAL_KEY, WHATS_NEW_MODAL_KEY } from '@/constants';
+import { N8nCallout, N8nHeading, N8nIcon, N8nLink, N8nMarkdown, N8nText } from '@n8n/design-system';
+import { createEventBus } from '@n8n/utils/event-bus';
+import { useVersionsStore } from '@/stores/versions.store';
+import { computed, ref, watch } from 'vue';
+import { useTelemetry } from '@/composables/useTelemetry';
+import { usePageRedirectionHelper } from '@/composables/usePageRedirectionHelper';
+import { useUIStore } from '@/stores/ui.store';
+
+const props = defineProps<{
+	modalName: string;
+	data: {
+		articleId: number;
+	};
+}>();
+
+const pageRedirectionHelper = usePageRedirectionHelper();
+
+const scroller = ref<RecycleScrollerInstance>();
+
+const i18n = useI18n();
+const modalBus = createEventBus();
+const versionsStore = useVersionsStore();
+const uiStore = useUIStore();
+const telemetry = useTelemetry();
+
+const nextVersions = computed(() => versionsStore.nextVersions);
+
+const openUpdatesPanel = () => {
+	uiStore.openModal(VERSIONS_MODAL_KEY);
+};
+
+const onUpdateClick = async () => {
+	telemetry.track('User clicked on update button', {
+		source: 'whats-new-modal',
+	});
+
+	await pageRedirectionHelper.goToVersions();
+};
+
+watch(
+	() => versionsStore.whatsNewArticles,
+	(whatsNewArticles) => {
+		for (const article of whatsNewArticles) {
+			if (!versionsStore.isWhatsNewArticleRead(article.id)) {
+				versionsStore.setWhatsNewArticleRead(article.id);
+			}
+		}
+	},
+	{ immediate: true },
+);
+
+modalBus.on('opened', () => {
+	const articleIndex = versionsStore.whatsNewArticles.findIndex(
+		(article) => article.id === props.data.articleId,
+	);
+
+	scroller.value?.scrollToItem(articleIndex);
+});
+</script>
+
+<template>
+	<Modal
+		max-width="860px"
+		max-height="85vh"
+		:event-bus="modalBus"
+		:name="WHATS_NEW_MODAL_KEY"
+		:center="true"
+		:show-close="false"
+	>
+		<template #header>
+			<div :class="$style.header">
```

Comment:
Spacing and alignment look a bit off to me here. Would maybe look better if bell icon is top-aligned
![image](https://github.com/user-attachments/assets/b5ec610d-7595-4546-9c98-d267c18c81de)

Reply:
I was hoping to get some designs for this but didn't, and this was left at my placeholder implementation :D Nik had some comments on this too so I'll discuss with him tomorrow on what we wnat this to look like.